lib/generators/my_zipcode_gem/models_generator.rb

```diff
@@ -3,19 +3,20 @@ class ModelsGenerator < Base
     include Rails::Generators::Migration
 
     source_root File.expand_path('../templates', __FILE__)
-    
+
     def initialize(*args, &block)
       super
+      migration_template 'migration.rb', "db/migrate/create_my_zipcode_gem_models.rb"
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.